cmd/template/dbdriver/files/service/scylla.tmpl

```diff
@@ -0,0 +1,192 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
```

Comment:
I do believe it should also be https://github.com/scylladb/gocql

Reply:
AFAIK we have to replace one module for another. 

Check [Gemini go.mod](https://github.com/scylladb/gemini/blob/master/go.mod) for a reference.